teps/0031-tekton-bundles-cli.md

```diff
@@ -66,3 +71,95 @@ Provide a way for users to...
 * Users may publish to any repository they have access to.
 
 * Only valid Tekton Bundles are produced.
+
+## Proposal
+
+Amend the Tekton CLI to provide the following commands (more detail later):
+
+```shell
+# REF = Docker-like Tekton Bundle reference, eg. docker.io/myworkspace/mybundle:1.0
+# BUNDLE_OBJECT = path to a file or directory or STDIN of a valid Tekton Task or Pipeline object to be included in the
+# bundle.
+
+# Generates a Tekton Bundle from a set of 1+ bundle objects. Like `docker`, tags the object with the specified reference
+# locally.
+tkn remote build <REF> [BUNDLE_OBJECT...]
```

Comment:
Should this be called `bundle`? That better lines up with the name of this feature.

Also I would confuse `tkn remote build` with the act of running a build remotely

Reply:
I tend to agree, we may want to refer to `bundle` here

Reply:
👍

Reply:
Sounds good to me.